internal/artifacts/project_test.go

```diff
@@ -53,14 +53,14 @@ var _ = Describe("Project", func() {
 			Ω(os.RemoveAll(getTestPath("result"))).Should(Succeed())
 		})
 		It("Sanity", func() {
-			err := ExecBuild("Makefile_tmp.mta", getTestPath("mta_with_zipped_module"), getResultPath(), "", "", "cf", true, os.Getwd, func(strings [][]string) error {
+			err := ExecBuild("Makefile_tmp.mta", getTestPath("mta_with_zipped_module"), getResultPath(), "", "", "cf", true, os.Getwd, func(strings [][]string, b bool) error {
 				return nil
 			}, true)
 			Ω(err).Should(Succeed())
 			Ω(filepath.Join(getResultPath(), "Makefile_tmp.mta")).ShouldNot(BeAnExistingFile())
```

Comment:
it does not relate to your change but as I remember ExecBuild does not create Makefile_tmp.mta but a file with some prefix and removes it at the end. So this check is not relevant and to be removed.

Reply:
It creates a file with the name it receives as first parameter, which is passed here as `"Makefile_tmp.mta"`. That's why it's checked at the end.